src/style/themes/default/antd/tootltip.less

```diff
@@ -0,0 +1,5 @@
+.@{tooltip-prefix-cls} {
+  &-inner {
+    color: @tooltip-color;
+  }
+}
```

Comment:
why is this necessary? since you already settle value of the variable, the theme should pick it up.

Reply:
good point!